guides/common/modules/con_performing-post-upgrade-tasks.adoc

```diff
@@ -4,6 +4,9 @@
 Some of the procedures in this section are optional.
 You can choose to perform only those procedures that are relevant to your installation.
 
+If you use custom templates created from a cloned default template, create the custom template again after the upgrade.
+Sometimes the default templates are changed during the upgrade, this will ensure that the custom templates are compatible with the latest version.
```

Comment:
```suggestion
The default templates are changed during the upgrade, this will ensure that the custom templates are compatible with the latest version.
```
If we have clear data, we should not use this at all. 
If this is less frequent, we can opt for `occasionally` instead of sometimes, according to SG.